packages/components/psammead-paragraph/src/index.stories.jsx

```diff
@@ -13,4 +14,18 @@ storiesOf('Components|Paragraph', module)
       <Paragraph script={script}>{paragraph}</Paragraph>
     )),
     { notes, knobs: { escapeHTML: false } },
+  )
+  .add(
+    'containing an inline link',
+    inputProvider(
+      [{ name: 'Paragraph' }, { name: 'Inline link' }],
+      ([paragraph, linkText], script) => (
+        <Paragraph script={script}>
+          {`${paragraph} `}
+          <InlineLink href="https://www.bbc.com">{linkText}</InlineLink>.
```

Comment:
```suggestion
          <InlineLink href="https://www.bbc.com">{linkText}</InlineLink>
```

remove hardcoded fullstop